kokoro/ubuntu/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+# create a timestamp file
+date +%s > timestamp.txt
+
+cd git/kokoro-codelab-elharo
```

Comment:
This was copped directly from the codelab and this directory likely wont exist in your actual repo.